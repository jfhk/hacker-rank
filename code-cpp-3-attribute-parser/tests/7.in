4 4
<tag1 value = "<HelloWorld>" okay = "</Flag>">
<tag2 name = "</Name1">
</tag2>
</tag1>
tag1.tag2~name
tag1~name
tag1~value
tag1~okay
