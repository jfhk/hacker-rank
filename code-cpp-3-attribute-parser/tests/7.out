</Name1
Not Found!
<HelloWorld>
</Flag>
